serialize.c

```diff
@@ -21,8 +22,6 @@
  * total: 6 Bytes
  */
 
-/* We only read Install Expr messages.
- */
```

Comment:
Was this comment not true?

Reply:
Ah good catch, I think this should have been updated, not deleted. Just did it now :)